lib/contentTypeParser.js

```diff
@@ -13,7 +13,8 @@ const {
   kBodyLimit,
   kRequestPayloadStream,
   kState,
-  kTestInternals
+  kTestInternals,
+  kReplyisError
```

Comment:
Typo, the `i` should be uppercase. I guess the test are not covering this.

Reply:
Good catch.

Reply:
Can you add a test to cover this file as well?

Reply:
Can you give me an idea how to do that? Isn't this beyond the scope of tests necessary for this PR? I would argue that such tests should then be added for every symbol importation in every file, not just this one.

Reply:
I think it's important to cover this case as well. Especially because there was a typo a nothing failed, which should not happen :D

Reply:
I understand that, but how can you test this? And if you test for this, why not test for all symbols in all files?

Reply:
What I mean is that we should add a test that triggers those errors, so we can validate that the symbol is doing what is expected from it.

Reply:
I added a test that addresses this part.